bindings/rust/lib.rs

```diff
@@ -0,0 +1,52 @@
+//! This crate provides Solidity language support for the [tree-sitter][] parsing library.
```

Comment:
```suggestion
//! This crate provides Solidity language support for the [tree-sitter][] parsing library.
//!
//! The bindings in https://github.com/tree-sitter/tree-sitter-javascript/blob/master/bindings/rust were used
//! as a template for the tree-sitter-solidity rust bindings.
//!
```

This is for future me 😄